metloom/variables.py

```diff
@@ -102,55 +107,161 @@ class CdecStationVariables(VariableBase):
-    RH = SensorDescription("RHUMV", "RELATIVE HUMIDITY", "RELATIVE HUMIDITY")
-    STREAMVOLUMEOBS = SensorDescription(
-        "SRVO", "STREAM VOLUME OBS", "STREAM VOLUME OBS"
-    )
-    STREAMVOLUMEADJ = SensorDescription(
-        "SRVOX", "STREAM VOLUME ADJ", "STREAM VOLUME ADJ"
-    )
+# Available sensors from Snotel
+SnotelVariables = make_dataclass(
+    "SnowtelVariables",
+    [
+        (
+            "SNOWDEPTH",
+            SensorDescription,
+            field(default=SensorDescription("SNWD", "SNOWDEPTH")),
+        ),
+        ("SWE", SensorDescription, field(default=SensorDescription("WTEQ", "SWE"))),
+        (
+            "TEMP",
+            SensorDescription,
+            field(default=SensorDescription("TOBS", "AIR TEMP")),
+        ),
+        (
+            "TEMPAVG",
+            SensorDescription,
+            field(
+                default=SensorDescription(
+                    "TAVG", "AVG AIR TEMP", "AIR TEMPERATURE AVERAGE"
+                )
+            ),
+        ),
+        (
+            "TEMPMIN",
+            SensorDescription,
+            field(
+                default=SensorDescription(
+                    "TMIN", "MIN AIR TEMP", "AIR TEMPERATURE MINIMUM"
+                )
+            ),
+        ),
+        (
+            "TEMPMAX",
+            SensorDescription,
+            field(
+                default=SensorDescription(
+                    "TMAX", "MAX AIR TEMP", "AIR TEMPERATURE MAXIMUM"
+                )
+            ),
+        ),
+        (
+            "PRECIPITATION",
+            SensorDescription,
+            field(
+                default=SensorDescription(
+                    "PRCPSA", "PRECIPITATION", "PRECIPITATION INCREMENT SNOW-ADJUSTED"
+                )
+            ),
+        ),
+        (
+            "PRECIPITATIONACCUM",
+            SensorDescription,
+            field(
+                default=SensorDescription(
+                    "PREC", "ACCUMULATED PRECIPITATION", "PRECIPITATION ACCUMULATION"
+                )
+            ),
+        ),
+        # TODO for the SCAN network this appears to be "RHUM", we may need a new class
+        (
+            "RH",
+            SensorDescription,
+            field(
+                default=SensorDescription(
+                    "RHUMV", "Relative Humidity", "RELATIVE HUMIDITY"
+                )
+            ),
+        ),
+        (
+            "STREAMVOLUMEOBS",
+            SensorDescription,
+            field(
+                default=SensorDescription(
+                    "SRVO", "STREAM VOLUME OBS", "STREAM VOLUME OBS"
+                )
+            ),
+        ),
+        (
+            "STREAMVOLUMEADJ",
+            SensorDescription,
+            field(
+                default=SensorDescription(
+                    "SRVOX", "STREAM VOLUME ADJ", "STREAM VOLUME ADJ"
+                )
+            ),
+        ),  # SensorDescription("WTEQ", "SWE"),
+    ]
+    + [
```

Comment:
This isi what flake8 is complaining about (and the other lines below like it). I think if this is `] + [` on one line it won't complain